changelog/updates/2023-08-02-2023-07-17-weekly.md

```diff
@@ -0,0 +1,5 @@
+- libxml2 ([2.11.4](https://gitlab.gnome.org/GNOME/libxml2/-/releases/v2.11.4))
+- libxslt ([1.1.38](https://gitlab.gnome.org/GNOME/libxslt/-/releases/v1.1.38))
+- hwdata ([0.372](https://github.com/vcrhonek/hwdata/releases/tag/v0.372))
```

Comment:
This is actually still at 0.371 (0.372 has unstable keywords), so should be dropped from changelog.